Cargo.toml

```diff
@@ -2,13 +2,13 @@
 
 name = "logger"
 version = "0.1.0"
-authors = ["Jonathan Reem <jonathan.reem@gmail.com>", "Michael Reinhard <mcreinhard@gmail.com>"]
+authors = ["Alexander Irbis <irbis.labs@gmail.com>", "Jonathan Reem <jonathan.reem@gmail.com>", "Michael Reinhard <mcreinhard@gmail.com>"]
 description = "Logging middleware for the Iron framework."
 repository = "https://github.com/iron/logger"
 keywords = ["iron", "web", "logger", "log", "timer"]
 license = "MIT"
 
 [dependencies]
 iron = { version = "0.4", default-features = false }
-term = "0.4"
+log = "*"
```

Comment:
The current version of [log](https://crates.io/crates/log) is `0.3.6`. We can't use `*` specifiers in published crates so this needs to be changed.